src/alarms/alarmgroups/PaginationBar.js

```diff
@@ -16,12 +16,12 @@ class PaginationBar extends Component {
           {links.map((link, i) => {
             const linkPlusOne = link + 1;
             if (page === linkPlusOne) {
-              return <div key={i}>{linkPlusOne}</div>;
+              return <div key={linkPlusOne}>{linkPlusOne}</div>;
             }
```

Comment:
Hier als key "linkPlusOne" (dit is het pagina nummer) gebruiken zal wel goed gaan, maar is feitelijk ook onjuist.
Namelijk de ene eerste pagina is de andere eerste pagina niet.
Correcter zou zijn om als key te gebruiken: filter + page , maar de filter is binnen dit component niet in scope.

Reply:
Maar binnen die lijst is de ene eerste pagina de andere toch wel?